stdlib.ispc

```diff
@@ -1083,6 +1122,7 @@ __declspec(safe) static inline uniform unsigned int64 reduce_max(unsigned int64
 REDUCE_EQUAL(int32, int32, IntMaskType)
 REDUCE_EQUAL(unsigned int32, int32, UIntMaskType)
 REDUCE_EQUAL(float, float, IntMaskType)
+REDUCE_EQUAL(float16, half, IntMaskType)
```

Comment:
Hope this is correct.

Reply:
It's correct if and only if it's covered with tests that work ;-)